.stylua.toml

```diff
@@ -0,0 +1,6 @@
+column_width = 120
```

Comment:
maybe we should start keeping these files in [config/](https://github.com/neovim/neovim/tree/master/config) ?

Reply:
Yes, I didn't like that it clobbered the root either. @tjdevries and I have been informally using 120, but we can also start being stricter on the lua code (or less strict on the C code) pending this discussion :)